app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from functools import reduce
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    return reduce(lambda accum,
+        player: accum + player.get_rating(), players, 0)
```

Comment:
The lambda function in `reduce` is correctly implemented to sum the ratings of players. Ensure that all player classes have the `get_rating` method implemented as expected.